testutil/environment/clean.go

```diff
@@ -19,18 +19,18 @@ import (
 // depend on each others.
 func (e *Execution) Clean(t testing.TB) {
 	t.Helper()
-	client := e.APIClient()
+	apiClient := e.APIClient()
```

Comment:
I later saw we could even access `e.client` directly; looks like `e.APIClient()` does nothing (no locks or anything).

I'll probably have more to do for this package, so may include such changes later.